toolchains/workspace-pnpm/run_audit.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+"""
+Runs a program in a directory.
```

Comment:
Probably oversight - not a good comment.

Reply:
Thanks, yup was from `run_in_dir.py` copy-paste